crates/sui/src/validator_commands.rs

```diff
@@ -587,6 +618,9 @@ impl Display for SuiValidatorCommandResponse {
             SuiValidatorCommandResponse::ReportValidator(response) => {
                 write!(writer, "{}", write_transaction_response(response)?)?;
             }
+            SuiValidatorCommandResponse::SerializedPayload(response) => {
+                write!(writer, "Serialized payload: {}", response)?;
+            }
```

Comment:
@joyqvq do we need to showcase a blake2 output as well?
If users use our `sign` function this is already performed internally, but if they somehow use external libs, then the serialized payload should be blake2b-256 first, then sign that.

Reply:
discussed offline authority sig do not commit to blake2b